### DISCUSSION TITLE
Strumeline.hx Question

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#421] [Q&A] [answered]
when this
![image](https://github.com/user-attachments/assets/7a820b22-cdc1-45ff-80c6-41ce41ef10c6)

**Comment by NexIsDumb** [comment, marked as answer]
we're handling that

**Reply by THEkatinamicrowave** [reply]
epic 
thank you king 😁😁😁😁
